scout/server/blueprints/variant/controllers.py

```diff
@@ -365,10 +365,19 @@ def observations(store, loqusdb, case_obj, variant_obj):
         if user_institutes_ids.isdisjoint(other_institutes):
             # If the user does not have access to the information we skip it
             continue
-        document_id = parse_document_id(chrom, str(pos), ref, alt, var_type, case_id)
-        other_variant = store.variant(document_id=document_id)
+
+        other_variant = store.variant(
+            case_id=other_case["_id"], document_id=variant_obj["variant_id"]
+        )
+
+        # IF variant is SV variant, look for variants with different sub_category occurring at the same coordinates
```

Comment:
Hm, why so? Are you thinking like BND vs DEL for different callers?

Reply:
This could happen, though e.g. DEL and DUP at the same pos are definately distinct (and would not normally occur in the same case or in a case with a match, so probably safe). More importantly, the start and end coordinates do not have to match exactly, but just be in a vicinity, with the exact interval depending on the size of the variant (rules in loqusdb). In these cases it would be good to return a link to the matched case, even if the variant was not found.

Reply:
No you are right, looking at the loqus code, the query if performed only on variants with the same sub-category. I'll fix! I've checked the rules in loqus (as you suggested) and I see how the query is runned now (not exact match, whoch makes a lot of sense for SVs). I might end up with a slightly different solution but I'm aware of the thing now!